straxen/contexts.py

```diff
@@ -124,7 +127,14 @@ def xenonnt_som(cmt_version="global_ONLINE", xedocs_version=None, _from_cutax=Fa
         cmt_version=cmt_version, xedocs_version=xedocs_version, _from_cutax=_from_cutax, **kwargs
     )
     del st._plugin_class_registry["peaklet_classification"]
-    st.register(straxen.PeakletClassificationSOM)
+    st.register(
+        (
+            straxen.PeakletClassificationSOM,
+            straxen.PeaksSOM,
+            straxen.PeakBasicsSOM,
+            straxen.EventBasicsSOM,
```

Comment:
These are new plugins which extend the already existing classical versions. With this context we will have a new lineage for everything above peaklets. The same data-type names as for the classical case are used (peaklet_classification, peaks, peak_basics, event_basics), such that we can use the standard processing procedure. 

* `PeakletClassificationSOM` computes first the classical classification before adding the SOM information.
* `PeaksSOM` propagates the information to Peaks such that we can propagate it to peak basics in  `PeakBasicsSOM`. 
* `EventBasicsSOM` is the same as the classical EventBasics, but also extended by the additional SOM fields.